src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -338,29 +338,20 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/DocumentTypes"
-  "/lookup/courtInformation/{courtName}":
+  "/lookup/courtLocations/":
```

Comment:
I would just go with "/court" court will be the resource

Reply:
and add a querystring parameter ?level=P / S

Reply:
Updated